infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -99,14 +89,26 @@ public LogReplicationConfigManager(CorfuRuntime runtime, String pluginConfigFile
         this.corfuStore = new CorfuStore(runtime);
 
         initStreamNameFetcherPlugin();
-        openTables();
+        openVersionTable();
+        // TODO: delete this method after upgrading
+        unregisterStreamsNameTable(runtime);
+    }
+
+    // Temporary helper method for remove Streams_Name_Table from registry table, for verticals
```

Comment:
For convention, lets keep method comments using doc comment which is set inside the comment delimiters /**...*/

Reply:
I wouldn't say it's temporal as this code is required as part of code evolution/backward compatibility. Also, this has nothing to do with verticals, the stream_name_table was created by LR and nothing to do with clients.

Reply:
If we are completely removing the table, it may be a good idea to take a look at 
https://github.com/CorfuDB/CorfuDB/pull/3230

to be sure we handle all corner cases.